mmv1/templates/terraform/examples/datastream_stream_mysql_gtid.tf.erb

```diff
@@ -0,0 +1,103 @@
+resource "google_sql_database_instance" "instance" {
```

Comment:
This test needs to referenced as an example. As is, this test file is not doing anything.

https://github.com/ravijaingit/magic-modules/blob/3697c8acbcd352a17953d607d053068799532cc4/mmv1/products/datastream/Stream.yaml#L57-L189

Reply:
done.